src/core/lib/event_engine/tcp_socket_utils.cc

```diff
@@ -70,6 +74,8 @@ absl::StatusOr<std::string> GetScheme(
       return "ipv6";
     case AF_UNIX:
       return "unix";
+    case AF_VSOCK:
```

Comment:
I believe this should be guarded by `#ifdef GRPC_HAVE_VSOCK`

Reply:
done